docs_new/introduction.md

```diff
@@ -0,0 +1,57 @@
+# IGUANA
```

Comment:
try to keep duplication as minimal as possible. Ideally this says them same as the README.md

In that case, it should be a symlink to README.md not a copy.